spec/requests/stories_index_spec.rb

```diff
@@ -215,5 +216,69 @@ def create_live_sponsor(org, tag)
       expect(response.body).to include("is sponsored by")
       expect(response.body).to include(sponsorship.blurb_html)
     end
+
+    context "with user signed in" do
+      before do
+        sign_in user
+      end
+
+      it "has mod-action-button" do
+        get "/t/#{tag.name}"
+        expect(response.body).to include('<a class="cta mod-action-button"')
+      end
+
+      it "does not render pagination" do
+        get "/t/#{tag.name}"
+        expect(response.body).not_to include('<span class="olderposts-pagenumber">')
+      end
+
+      it "does not render pagination even with many posts" do
+        create_list(:article, 20, user: user, featured: true, tags: [tag.name], score: 20)
+        get "/t/#{tag.name}"
+        expect(response.body).not_to include('<span class="olderposts-pagenumber">')
+      end
+    end
+
+    context "without user signed in" do
+      let(:tag) { create(:tag) }
+
+      it "does not render pagination" do
```

Comment:
I think these might be more clear if we grouped the tests by path `/t/<tag>` vs `/t/<tag>/page` since that is what causes the behavior change. Whether the user is signed in or not the behavior is often the same, for example:
- "with user signed in does not render pagination"
- "without user signed in does not render pagination"
- "with user signed in does not render pagination even with many posts"
- "without user signed in does not render pagination even with many posts"